main/admin/access_url_edit_courses_to_url.php

```diff
@@ -197,7 +197,7 @@ function remove_item(origin) {
                             <?php
                             foreach ($no_course_list as $no_course) {
                                 ?>
-                                <option value="<?php echo $no_course['id']; ?>"><?php echo $no_course['title'].' ('.$no_course['code'].')'; ?></option>
+                                <option value="<?php echo $no_course['id']; ?>" title="<?php echo $no_course['title'].' ('.$no_course['code'].')'; ?>"><?php echo $no_course['title'].' ('.$no_course['code'].')'; ?></option>
```

Comment:
You need to escape this. If there is a `"` into your course title, it will break the HTML (and the whole page).
I believe using htmlentities() should work for that. To test, make sure your course title has a `"` it is and check if it shows correctly with and without htmlentities().

Reply:
Fixed with htmlentities